.github/workflows/lint.yml

```diff
@@ -13,14 +13,14 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dprint/check@v2.2
         with:
-          dprint-version: '0.40.2' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
+          dprint-version: '' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
```

Comment:
Fixed in https://github.com/kachick/dotfiles/commit/8b6c3654f8bce66909edd8cf7990ee8d13a1c442